README.md

```diff
@@ -1,14 +1,39 @@
-# RainCatcher
 
-[![Build Status](https://travis-ci.org/feedhenry-raincatcher/raincatcher-core.svg?branch=master)](https://travis-ci.org/feedhenry-raincatcher/raincatcher-core) [![Coverage Status](https://coveralls.io/repos/github/feedhenry-raincatcher/raincatcher-core/badge.svg?branch=master)](https://coveralls.io/github/feedhenry-raincatcher/raincatcher-core?branch=master)
+![Logo](img/logo-blue.png)
+> Workforce management platform
 
-Feedhenry RainCatcher Core Repository
+[![Build Status](https://travis-ci.org/feedhenry-raincatcher/raincatcher-core.svg?branch=master)](https://travis-ci.org/feedhenry-raincatcher/raincatcher-core) [![Coverage Status](https://coveralls.io/repos/github/feedhenry-raincatcher/raincatcher-core/badge.svg?branch=master)](https://coveralls.io/github/feedhenry-raincatcher/raincatcher-core?branch=master)
 
 ## About RainCatcher
 
-## Developing
+RainCatcher is an Workforce Management platform with main focus on delivering set of modules and user interface for mobile developers trying to integrate WFM features into their application.
```

Comment:
We may add more industry specific info from here: 
https://en.wikipedia.org/wiki/Workforce_management

That's probably going to be added directly into docs